src/app/modules/shared/paginator-extentions/mat-paginator-intl-ukr.ts

```diff
@@ -0,0 +1,21 @@
+import {MatPaginatorIntl} from '@angular/material/paginator';
+
+export class MatPaginatorIntlUkr extends MatPaginatorIntl {
```

Comment:
Remove `ukr` from class and file name, because project won't have specific static language.
Those and other strings you may leave, I will change it later, when will be adding localization.
```
  itemsPerPageLabel = 'Записів на сторінці:';
  nextPageLabel = 'Наступна сторінка';
  previousPageLabel = 'Попередня сторінка';
```